src/TLuaInterpreter.cpp

```diff
@@ -429,12 +430,12 @@ int TLuaInterpreter::getProfileName( lua_State * L )
     return 1;
 }
 
-int TLuaInterpreter::raiseInterProfileEvent( lua_State * L )
+int TLuaInterpreter::sendGlobalEvent( lua_State * L )
```

Comment:
OK, I understand - The entire intent is to reduce confusion by the different name :) Definitely easier to use/type